src/components/Identity/customerIds/createCustomerIds.js

```diff
@@ -61,7 +62,9 @@ export default ({ eventManager, consent, logger }) => {
         const event = eventManager.createEvent();
         return consent.awaitConsent().then(() => {
           // FIXME: Konductor shouldn't require an event.
-          return eventManager.sendEvent(event);
+          // we're now returning an object at onResponse
+          // here we need to add 'then(noop)' because we are not returning a value
+          return eventManager.sendEvent(event).then(noop);
```

Comment:
@ninaceban Would you please update the tests accordingly to assert that we're not resolving the promise with a value?

Reply:
yes, sorry I forgot about these